resources/settings.xml

```diff
@@ -21,10 +21,10 @@
     <setting id="catchup_source" label="30210" type="enum" values="Any|Akamai|Limelight" default="0" enable="eq(-2,true)" />
     <setting id="catchup_bitrate" label="30211" type="enum" values="Fastest|0.8 Mbps|1.0 Mbps|1.5 Mbps|1.8 Mbps|2.4 Mbps|3.1 Mbps|5.5 Mbps" default="0" enable="eq(-3,true)" />
     <setting id="live_source" label="30220" type="enum" values="Any|Akamai|Limelight" default="0" enable="eq(-4,true)" />
-    <setting id="live_bitrate" label="30230" type="enum" values="Fastest|0.3 Mbps|0.5 Mbps|1.0 Mbps|1.5 Mbps|1.7 Mbps|2.1 Mbps|2.9 Mbs|3.6 Mbps|5.1 Mbps" default="5" enable="eq(-5,true)" />
+    <setting id="live_bitrate" label="30230" type="enum" values="Highest|576 Kbps|974 Kbps|1732 Kbps|3000 Kbps|5308 Kbps" default="0" enable="eq(-5,true)" />
```

Comment:
Please keep the original settings. This is something not to temper with. If somebody selected 3.6 Mbps before, they will just get a script error with the proposed changes. This is not user friendly at all.